ES/ESmy018.xml

```diff
@@ -32,86 +32,185 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-018</idno>
                </msIdentifier>
                <msContents>
-                  <summary>I) Taʾammǝra Māryām (fols. 3ra-43vb)&#xD; I-1) Introductory exhortation
+                  <summary>
+                    I) Taʾammǝra Māryām (fols. 3ra-43vb); I-1) Introductory exhortation
                      (fols. 3ra-4va)&#xD; I-2) Taʾammǝra Māryām: 30 miracles (fols. 4va-43vb)&#xD; II)
                      Discourse by John Chrysostom on the life and decollation of John the Baptist
-                     (fol. 44ra-60va) &#xD; III) Homily on the finding of the head of John the
-                     Baptist (fols. 60vb-65ra)&#xD; IV) Homily by John Chrisostom on the glory of
-                     John the Baptist (65ra-74ra)&#xD; &#xD; Additiones 1-2; colophons
-                     1-2.</summary>
-                  <msItem xml:id="ms_i1">
+                     (fol. 44ra-60va) &#xD; I</summary>
 
-                     <incipit>
-                        <foreign xml:lang="en"/>
-                     </incipit>
-                     <explicit>
-                        <foreign xml:lang="en"/>
-                     </explicit>
-                     <note/>
+                    <msItem xml:id="ms_i1">
+                     <locus from="3ra" to="43vb"/>
+                     <title type="complete" ref="LIT2384Taamme"/>
+
+                     <msItem xml:id="ms_i1.1">
+                      <locus from="3ra" to="4va"/>
+                      <title type="complete" ref="LIT3584Intro"/>
+                      <incipit xml:lang="gez">
+                         <locus target="#1r"/><hi rend="rubric">ስምዑ፡ እነግረክሙ፡ ኦአኃዊነ፡</hi> ውሉደ፡
+                         ቤተ፡ <sic>ክርስቲ</sic> እግዚአብሔር፡ ይባርክሙ፡ ወያብርህ፡ አዕይንተ፡ ልብክሙ፡ ለሰሚዐ፡ ተአምሪሃ፡
+                         ለእግዝእትነ፡ <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ አምላክ፡ ዘኢይክሉ፡ ሰማያውያን፡ ወምራውያን፡ ፈጽሞ፡ ውዳሴሃ፡ ኢሰማያውያን፡ ወኢምድራውያን፡
+                      ጸሎታ፡ ወበረከታ፡ ይሀሎ፡ ምስለ፡ ገብሩ፡ ተክለ፡ እዎስጣቴዎስ፨ ዝውእቱ፡ ቅድመ፡ ተአምር፡ ይትነበብ፡
+                      </incipit>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.2">
+                       <locus from="3ra" to="4va"/>
+                       <title type="complete" ref="LIT3584Intro"/>
+                     <note>Miracles</note>
                   </msItem>
-                  <!--The data in the items below refers to other items. please remove this items and move the information to the appropriate place.-->
-                  <msItem xml:id="ms_i2">
+                  <msItem xml:id="ms_i1.">
```

Comment:
remove the msItem, or I don't understand the structure at all.

it must be `<msItem xml:id="ms_i1">` for the whole complex of miracles LIT2384Taamme, underneath you can have
1.1 Intro LIT3584Intro
1.2 Miracles LIT3585Taamme
1.2.1 Miracle 1
1,2,2 Miracle 2 etc. to 
1.2.30 Miracle 30

then you have 
`<msItem xml:id="ms_i2">` Discourse
`<msItem xml:id="ms_i3">` Homily
`<msItem xml:id="ms_i4">` Homily